src/common/components/SelectSearch.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import Select from "./Select";
+
+export default class SelectSearch extends Component {
+    componentDidMount() {
+        $(document).ready(() => {
```

Comment:
This is an extra check, the component's DOM will always be ready for this moment.